src/java.base/share/classes/sun/security/util/KeyStoreDelegator.java

```diff
@@ -284,7 +289,18 @@ public void engineLoad(InputStream stream, char[] password)
             }
 
             if (debug != null) {
-                debug.println("Loaded a keystore in " + type + " format");
+                if (stream instanceof FileInputStream) {
+                    String keystorePath = SharedSecrets
+                                    .getJavaIOFileInputStreamAccess()
+                                    .getPath((FileInputStream) stream);
+                    if (keystorePath != null) {
```

Comment:
if `keystorePath` is null, no "Loaded" statement is printed. Let's cover that case also.